File: client/dom/scripts/all_tests.py
#!/usr/bin/python
# Copyright (c) 2011, the Dart project authors.  Please see the AUTHORS file
# for details. All rights reserved. Use of this source code is governed by a
# BSD-style license that can be found in the LICENSE file.

"""This entry point runs all script tests."""

import logging.config
import unittest

if __name__ == '__main__':
  logging.config.fileConfig('logging.conf')
  suite = unittest.TestLoader().loadTestsFromNames([
      'templateloader_test',
      'pegparser_test',
      'idlparser_test',
      'idlnode_test',
      'idlrenderer_test',
      'database_test',
      'databasebuilder_test',
      'emitter_test',
      'dartgenerator_test',
      'multiemitter_test'])
  unittest.TextTestRunner().run(suite)

File: pkg/polymer_expressions/test/all_tests.dart
// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
// for details. All rights reserved. Use of this source code is governed by a
// BSD-style license that can be found in the LICENSE file.

library all_tests;

import 'eval_test.dart' as eval;
import 'parser_test.dart' as parser;
import 'tokenizer_test.dart' as tokenizer;
import 'visitor_test.dart' as visitor;

main() {
  eval.main();
  parser.main();
  tokenizer.main();
  visitor.main();
}
